### DISCUSSION TITLE
The value of el-select in el-popover changed will close el-popover.

### DISCUSSION THREAD

**Discussion by cathino** [element-plus/element-plus#12379] [Q&A]
My situation is that a drop-down menu contains several components. After clicking, the dialog box in the component is displayed, which contains several selects, but after the value of the select is modified, the entire drop-down menu will not be displayed.
I tried `el-dropdown` and `el-popover`, in comparison, `el-popover` works better

A simple demo like:

```vue
<el-popover>
    <el-dialog>
        <el-select>
          <el-option> value1 </el-option>
          <el-option> value2 </el-option>
        </el-select>
    <el-dialog>
</el-popover>
```

If change the vaule of el-select, el-popover will be closed.
Can I change the behaviour?

**Comment by cathino** [comment]
The behaviour of `el-select-v2` is as same as `el-select`.

**Reply by serkon** [reply]
https://github.com/element-plus/element-plus/issues/10360

disable teleport on `el-select` using `<el-select :teleport='false'>`
